### PR TITLE
feature: SYM-3431: badge component

### PR DESCRIPTION
**Pull request by matthieucrouzet-sym**

### CODE REVIEW

File: src/components/misc/badge/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  border: none;
+  padding: 4px 8px;
+`;
+
+const IconWrapper = styled.span`
+  margin-left: 8px;
+  &:hover {
+    cursor: pointer;
+  }
+`;
```
Comment (AnthonyDS-symphony) [inline src/components/misc/badge/index.js:22]: You should create a `theme.js` and put them inside. :)

**Review by AnthonyDS-symphony — CHANGES_REQUESTED** [review]
LGTM!

**Comment by romain-guillot-symphony** [comment]
I think the current base branch is `feature/sparc-eq-derivatives`

**Comment by matthieucrouzet-sym** [comment]
> I think the current base branch is `feature/sparc-eq-derivatives`

https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit/pull/128
I change the PR

**Pull request closed without merging**